Makefile

```diff
@@ -2,6 +2,16 @@
 Q=$(if $V,,@)
 SRC=$(shell find . -type f -name '*.go')
 
+# protoc-gen-go constraints
+GEN_GO_BIN ?= protoc-gen-go
+GEN_GO_MIN_VERSION ?= 1.31.0
+GEN_GO_VERSION ?= $(shell $(GEN_GO_BIN) --version | awk --field-separator=' v' '{print $$NF}')
+
+# protoc-gen-go-grpc constraints
+GEN_GRPC_BIN ?= protoc-gen-go-grpc
+GEN_GRPC_MIN_VERSION ?= 1.3.0
+GEN_GRPC_VERSION ?= $(shell $(GEN_GRPC_BIN) --version | awk --field-separator=' ' '{print $$NF}')
```

Comment:
```suggestion
GEN_GRPC_VERSION ?= $(shell $(GEN_GRPC_BIN) --version | awk -F ' ' '{print $$NF}')
```

Reply:
weird, I'm on ubuntu myself and the action's also running on ubuntu in CI.